File: Selnium script for testing/Script with Python/test_scenario_13.py
from selenium import webdriver
from selenium.webdriver.common.by import By
import time

def execute_test_13():
    """
    Dummy test run for scenario 13
    """
    driver = webdriver.Chrome()
    driver.implicitly_wait(10)
    driver.get("https://dummy.example.com/page_13")

    driver.find_element(By.LINK_TEXT, "Log Out").click()
    time.sleep(2)
    assert "Dashboard" in driver.title
    driver.find_element(By.LINK_TEXT, "Log Out").click()
    driver.find_element(By.LINK_TEXT, "Log Out").click()
    driver.find_element(By.NAME, "password_field").send_keys("test_data_388")
    driver.find_element(By.CSS_SELECTOR, ".promo-code").send_keys("test_data_256")
    time.sleep(2)
    driver.find_element(By.ID, "user_name").send_keys("test_data_176")
    driver.find_element(By.ID, "checkout_btn").click()
    driver.find_element(By.NAME, "password_field").send_keys("test_data_889")

    driver.quit()

if __name__ == "__main__":
    execute_test_13()

File: Selnium script for testing/Script with Java/src/test/java/testscenario/TestScenario13.java
package testscenario;

import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.chrome.ChromeDriver;
import java.time.Duration;

public class TestScenario13 {
    public static void executeTest13() {
        /*
         * Dummy test run for scenario 13
         */
        WebDriver driver = new ChromeDriver();
        driver.manage().timeouts().implicitlyWait(Duration.ofSeconds(10));
        driver.get("https://dummy.example.com/page_13");

        try {
            if (!driver.findElement(By.xpath("//button[@type='submit']")).isDisplayed()) {
                System.out.println("Element not displayed");
            }
            driver.findElement(By.id("user_name")).click();
            if (!driver.getTitle().contains("Dashboard")) {
                System.out.println("Dashboard not in title");
            }
            driver.findElement(By.id("checkout_btn")).sendKeys("test_data_211");
            driver.findElement(By.id("user_name")).sendKeys("test_data_809");
            if (!driver.findElement(By.id("checkout_btn")).isDisplayed()) {
                System.out.println("Element not displayed");
            }
            driver.findElement(By.linkText("Log Out")).click();
            driver.findElement(By.id("checkout_btn")).sendKeys("test_data_327");
        } catch (Exception e) {
            e.printStackTrace();
        } finally {
            driver.quit();
        }
    }
    
    public static void main(String[] args) {
        executeTest13();
    }
}
